### PR TITLE
Extended the Requirements Specification.

### PR DESCRIPTION
**Pull request by m1cm1c**

### CODE REVIEW

**Comment by m1cm1c** [comment]
I kicked out some else's attempt to fix the spelling mistake in `Beagle can be used for the verifying of design and implementation.`, btw. Then I fixed it.

**Comment by flyfl** [comment]
Lgtm. But I can't comment on the failed integration tests as I am not aware of what is actually happening in the travis shell.

**Comment by m1cm1c** [comment]
It took me a long time to figure it out. I checked all files for errors and checked them with meld against the upstream because I thought that I made some mistake when merging them manually. But in the end it was just a use of a non-existent definition.

File: doc/Requirements Specification/Parts/Application.lyx [hunk 1]
```diff
@@ -238,7 +238,7 @@ name "/A40/"
 
 \end_inset
 
- Beagle can be used for the verifying of design and implementation.
+ Beagle can be used for the verification of design and implementation.
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Application.lyx]: I’d prefer “Beagle can be used to verify design and implementation”. Simpler. What do you think?

File: doc/Requirements Specification/Parts/Application.lyx [hunk 2]
```diff
@@ -399,5 +399,61 @@ reference "/A30/"
...
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/A110/"
+
+\end_inset
+
+ Server (optional), add possibility to run on Server as additional function.
+\end_layout
+
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Application.lyx]: Why did you reintroduce this part? @acBerger removed it on purpose!

File: doc/Requirements Specification/Parts/Terms and Definitions.lyx [hunk 3]
```diff
@@ -1359,6 +1359,54 @@ glspl{resource demand}.
...
+\end_layout
+
+\begin_layout Plain Layout
+
+	description={
+\end_layout
+
+\begin_layout Plain Layout
+
+		a software set containing a Java Virtual Machine, a browser plugin, the
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Terms and Definitions.lyx]: > a browser plugin

Not really. You can install the `openjdk` without `icedtea`, for example.

**Comment by jGleitz** [comment]
@m1cm1c Can you active Travis for your account? That would allow us to see the rendered PDF without checking out your PR. Contact me for assistance.

File: doc/Requirements Specification/Parts/Application.lyx (hunk 1 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Application.lyx]: Much better.

File: doc/Requirements Specification/Parts/Terms and Definitions.lyx (hunk 3 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Terms and Definitions.lyx]: It probably doesn't count as a full JRE, then. Not my idea, I just looked up what it contains and wrote it down. I didn't find it on the official website, but it says pretty much the same: https://java.com/en/download/faq/whatis_java.xml

**Comment by jGleitz** [comment]
@m1cm1c Thanks

**Comment by jGleitz** [comment]
[See this PR’s rendered PDF](http://m1cm1c.github.io/Beagle/Requirements%20Specification.pdf)

**Comment by acBerger** [comment]
lgtm

**Comment by RomanLangrehr** [comment]
Lgtm.

**Pull request merged**